charts/erzwingungshaft/README.md

```
@@ -0,0 +1,23 @@
+```bash
+helm upgrade --install -f values.yaml erzwingungshaft charts/erzwingungshaft
+```
+
+The command deploys cronjob and additional configMap to start erzwingungshaft on kubernetes cluster. The [Configuration](#configuration) section lists the parameters that can be configured during installation. Use values.yaml xjustiz.modules.applicationYML to configure erzwingungshaft application.yaml.
+
+## Configuration
+
+| Key                                        | Type   | Default          | Description                                                                                                                                                                                                                                                |
+| ------------------------------------------ | ------ | ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| modules.eai.cronjob.schedule                           | string | - | Valid crontab expression to start kubernetes cronjob                                                                                                                                                                                                                                 |
+| modules.eai.applicationYML                                  | string | -             | application.yaml containing https://github.com/it-at-m/Erzwingungshaft/blob/main/erzwingungshaft-eai/src/main/resources/application.yml.                                                                                                                                                                                                                                                 |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)